src/modules/legends.js

```diff
@@ -0,0 +1,19 @@
+export const LEGEND_DISPLAY_STRATEGY_BY_DATA_ITEM = 'BY_DATA_ITEM'
```

Comment:
The first place where I added these is the `LegendDisplayStrategy` and `LegendDisplayStyle` options in DV app.
Since they are used in many places now it makes sense to import them from here.
Once this is merged I can update the option components in DV app.